mongodb/plan.sh

```diff
@@ -0,0 +1,57 @@
+pkg_name=mongodb
+pkg_origin=core
+pkg_version=3.2.9
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="High-performance, schema-free, document-oriented database"
+pkg_license=('AGPL-3.0')
+pkg_source=http://downloads.mongodb.org/src/${pkg_name}-src-r${pkg_version}.tar.gz
+pkg_shasum=25f8817762b784ce870edbeaef14141c7561eb6d7c14cd3197370c2f9790061b
+pkg_filename=${pkg_name}-src-r${pkg_version}.tar.gz
+pkg_dirname=${pkg_name}-src-r${pkg_version}
+pkg_deps=(core/gcc-libs core/glibc core/openssl)
+pkg_build_deps=(
+  core/coreutils
+  core/gcc
+  core/glibc
+  core/patchelf
```

Comment:
The build works without `core/patchelf`.
I want to note that trying to build with `DO_CHECK=true` fails, saying `ImportError: No module named pymongo`.